crates/holochain/src/core/queue_consumer.rs

```diff
@@ -194,7 +197,8 @@ impl TriggerSender {
     /// inconsistency from the perspective of any particular CPU thread
     pub fn new() -> (TriggerSender, TriggerReceiver) {
         let (tx, rx) = mpsc::channel(num_cpus::get());
-        (TriggerSender(tx), TriggerReceiver(rx))
+        let waker = futures::task::noop_waker();
```

Comment:
@maackle FYI we need `Waker` to create a `Context` to use the `poll_recv` function. I'm not sure if this dummy waker will serve us well. Once the tests compile we'll find out :wink:

Reply:
CI seems to still have some errors referencing tokio 0.2

```
thread 'thread 'core::ribosome::host_fn::call::wasm_test::call_the_same_cellFAILEDthread '' panicked at 'core::ribosome::host_fn::call_remote::wasm_test::call_remote_test
 not currently running on a Tokio 0.2.x runtime.core::ribosome::host_fn::capability_grants::wasm_test::ribosome_authorized_call' panicked at ' ', ' panicked at 'not currently running on a Tokio 0.2.x runtime.20'
```